github/table_github_user.go

```diff
@@ -10,6 +10,42 @@ import (
 	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
 )
 
+func gitHubUserColumns() []*plugin.Column {
```

Comment:
Do we still need this change if we're not using these columns in the `github_team_member` table anymore?

Reply:
It makes it more consistent, but technically not indeed. I'll revert it 👍